devel/qt5pas/Portfile

```diff
@@ -27,6 +27,9 @@ checksums           rmd160  878d64c1e5a7c0842d51d3f37d2644a00d32fdb5 \
 
 worksrcdir          lazarus/lcl/interfaces/qt5/cbindings
 
+# Workaround xcrun: error: SDK "macosx12" cannot be located
+configure.sdk_version
```

Comment:
Doesn't this mean that if a user requests a specific SDK in their macports.conf this will no longer be respected?

Reply:
I'm not sure what is a proper way to fix it, but I've just found this workaround across macports:
```sh
ag "Workaround xcrun: error: SDK \"macosx12\" cannot be located"
devel/iaito/Portfile
40:# Workaround xcrun: error: SDK "macosx12" cannot be located

devel/qt5pas/Portfile
30:# Workaround xcrun: error: SDK "macosx12" cannot be located

databases/sqlitebrowser/Portfile
60:    # Workaround xcrun: error: SDK "macosx12" cannot be located

net/qBittorrent-qt5/Portfile
89:# Workaround xcrun: error: SDK "macosx12" cannot be located

editors/liteide/Portfile
38:# Workaround xcrun: error: SDK "macosx12" cannot be located

math/qalculate-qt/Portfile
29:# Workaround xcrun: error: SDK "macosx12" cannot be located

graphics/QtPBFImagePlugin/Portfile
29:# Workaround xcrun: error: SDK "macosx12" cannot be located
```

Reply:
Then my question applies to all of those ports as well.